src/App.tsx

```diff
@@ -1,14 +1,58 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+import './App.scss';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { Todo } from './types/Todo';
+import { getTodos } from './api';
+import { StatusFilter } from './helpers/StatusFilter';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [loader, setLoader] = useState(false);
+  const [inputValue, setInputValue] = useState('');
+  const [todoFilter, setTodoFilter] = useState<StatusFilter>(StatusFilter.All);
+  const [selectedTodoId, setSelectedTodoId] = useState<number | null>(null);
+
+  const visibleTodos = todos.filter(todo => {
+    const matchesQuery = todo.title.toLowerCase().trim()
+      .includes(inputValue.toLowerCase().trim());
```

Comment:
let's create separate variable for `todo.title` and `inputValue`

```suggestion
    const formattedTitle = todo.title.toLowerCase().trim()
    const formattedInputValue = inputValue.toLowerCase().trim()
    
    const matchesQuery = formattedTitle.includes(formattedInputValue);
```